### PR TITLE
Add JuMPdev content

### PR DESCRIPTION
**Pull request by odow**
I've uploaded the slides to the NumFOCUS drive, but I still need to change the links in the text.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+in conjunction with UCLouvain.
+
+Further details, including a call for speakers, will be announced shortly.
+
+For information on the previous JuMP-dev workshops, see:
+[2019](meetings/santiago2019/), [2018](meetings/bordeaux2018/), [2017](meetings/mit2017/).
+
+---
+
+- **Overview**: [presentation](https://www.youtube.com/watch?v=7LNeR299q88) and [workshop](https://www.youtube.com/watch?v=nnL7yLMVu6c) from JuliaCon 2015
```
Comment (mlubin) [inline index.md]: The JuliaCon presentation and the IJOC paper are old material. If we list it we should make it clear that it's outdated. Not content for the main page.

File: index.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+Once you've installed JuMP, a good next step is the [Quickstart Guide](https://www.juliaopt.org/JuMP.jl/stable/quickstart/).
+
+There are also a number of tutorials available at [JuMPTutorials.jl](https://github.com/JuliaOpt/JuMPTutorials.jl).
+
+If you need help, search the [Forum](https://discourse.julialang.org/c/domain/opt)
+for similar questions, or open a new one.
+
+---
+
+##### 4th annual JuMP-dev
```
Comment (mlubin) [inline index.md]: This should be above "getting started"

Comment (mlubin) [inline index.md]: We should use heading depths correctly. `#####` is a sub-heading under `####`. If the issue is the font size of `##`, then that can be addressed with a change to the css or html.

File: pages/code.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Beneath JuMP, there is an abstraction layer called MathOptInterface ([source code](https://github.com/JuliaOpt/MathOptInterface.jl)). MathOptInterface defines an API that solvers
+should implement so that they can be used by JuMP, handles the automatic
+reformulation of problems via _bridges_, and has a large infrastructure for
+automatically testing solvers.
+
+At the bottom of the stack, there are solver wrappers. These packages wrap the
+solvers, which are often written in C or C++, and implement the API defined by
+MathOptInterface.
+
+A non-exhaustive list of solvers available through JuMP is given by the
```
Comment (mlubin) [inline pages/code.md]: I'm hesitant to have multiple copies of this table. What about linking to the table in the JuMP documentation for now?

File: assets/jump-logo-with-text-large.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...4.52-2,17.69a19.21,19.21,0,0,1-7.34,8.9q-4.86,3.11-12.34,3.11-8.13,0-13.13-3.62a19.27,19.27,0,0,1-7.06-10.12c-.94-3-1.45-8.44-1.45-16.32V50.57Zm138.87,0h13.46q11,0,15.76,2a16,16,0,0,1,7.63,6.54,20.21,20.21,0,0,1,2.79,10.8,18.23,18.23,0,0,1-3.65,11.61A18.82,18.82,0,0,1,363.86,88q-3.66,1-13.37,1v28.26H337.78V50.54Zm12.71,26.08h4a22.42,22.42,0,0,0,6.63-.68,5.82,5.82,0,0,0,2.92-2.25,6.53,6.53,0,0,0,1.07-3.79,6.09,6.09,0,0,0-3-5.62C360.69,63.41,358,63,354.06,63h-3.57V76.62Z"/><line class="cls-2" x1="19.64" y1="67.28" x2="120.16" y2="28.42"/><polyline class="cls-3" points="3.32 118.72 89.75 32.29 118.72 3.32"/><path class="cls-4" d="M106.29,120.18a17.17,17.17,0,1,1,12.16-5A17.06,17.06,0,0,1,106.29,120.18Z"/><path class="cls-5" d="M106.29,120.18a17.17,17.17,0,1,1,12.16-5A17.06,17.06,0,0,1,106.29,120.18Z"/><path class="cls-6" d="M106.29,79.33a17.19,17.19,0,1,1,12.16-5A17.07,17.07,0,0,1,106.29,79.33Z"/><path class="cls-7" d="M106.29,79.33a17.19,17.19,0,1,1,12.16-5A17.07,17.07,0,0,1,106.29,79.33Z"/><path class="cls-8" d="M65.28,120.18a17.19,17.19,0,1,1,12.15-5A17,17,0,0,1,65.28,120.18Z"/><path class="cls-9" d="M65.28,120.18a17.19,17.19,0,1,1,12.15-5A17,17,0,0,1,65.28,120.18Z"/></g></g></svg>
```
Comment (mlubin) [inline assets/jump-logo-with-text-large.svg]: I'm wondering if we need two copies of the logo checked in. It's an SVG so shouldn't be hard to rescale inline.

**Comment by odow** [comment]
Okay, I took a pass. Ready for another review.

File: _posts/2019-03-14-jump_dev_2019.md [hunk 5]
```diff
@@ -197,3 +197,19 @@ Ver la [versión en inglés](#schedule).
 - Diego Moran (Universidad Adolfo Ibáñez; Organización Local)
 
 Contactar a jump-dev-committee at googlegroups.com para más información.
+
+[joaquim_dias_garcia]: https://drive.google.com/file/d/1XKQQTPb5iHNYgwyU9mlrHzvq6kY8Hw7X/view?usp=sharing
+[tomas_lagos_gonzalez]: https://drive.google.com/file/d/1fVY6Wd6EMplax8QhR9y--ufEPqzRq_M5/view?usp=sharing
+[alessandro_soares]: https://drive.google.com/file/d/1NnF6HaIer6oCTrAWnkHb8B5uKxSTqyio/view?usp=sharing
+[andrew-rosemberg]: https://drive.google.com/file/d/19o4D1VPanZvQ8uNMgmr5sutRoQVyStcI/view?usp=sharing
```
Comment (mlubin) [inline _posts/2019-03-14-jump_dev_2019.md]: `andrew_rosemberg`

File: _posts/2019-03-14-jump_dev_2019.md [hunk 6]
```diff
@@ -197,3 +197,19 @@ Ver la [versión en inglés](#schedule).
 - Diego Moran (Universidad Adolfo Ibáñez; Organización Local)
 
 Contactar a jump-dev-committee at googlegroups.com para más información.
+
+[joaquim_dias_garcia]: https://drive.google.com/file/d/1XKQQTPb5iHNYgwyU9mlrHzvq6kY8Hw7X/view?usp=sharing
```
Comment (mlubin) [inline _posts/2019-03-14-jump_dev_2019.md]: The drive links to notebooks don't seem to work. We may have to commit those to the repository.

File: index.md [hunk 7]
```diff
@@ -30,6 +30,9 @@ There are also a number of tutorials available at [JuMPTutorials.jl](https://git
 If you need help, search the [Forum](https://discourse.julialang.org/c/domain/opt)
 for similar questions, or open a new one.
 
+There is also an excellent book, [Julia Programming for Operations Research](http://www.chkwon.net/julia/), which covers a variety of topics on optimization, with an emphasis on
```
Comment (mlubin) [inline index.md]: "excellent" is maybe too much of an official endorsement. We would also list other future books written about JuMP (assuming they're reasonably good). Perhaps "see also".

File: _posts/2019-03-14-jump_dev_2019.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+
+<tt>14:00 - 15:30</tt> **Advanced tutorial** [[Notebooks](https://github.com/blegat/JuMP-dev_2019_tutorial)] [[video](https://www.youtube.com/watch?v=gUf8kmdoWzo&list=PLP8iPy9hna6QkxelZiDOc5HnVGvQmEsLX&index=21&t=0s)]
+
+<tt>15:30 - EOD</tt> **Collaboration time and core developer meeting**
+
+### Thursday, March 14: JuMP-dev Day 3 at Campus San Joaquín ([Centro de Innovación, room "Sala 203"](https://goo.gl/maps/GVKuqnqdujS2))
+
+<tt>9:00 - 10:25</tt> **Algorithms**
+* Thuener Silva (PUC-Rio): _Solving Large-scale problems using JuMP_ [[slides][thuener_silva]] [[video](https://www.youtube.com/watch?v=CUCRie3udHI&list=PLP8iPy9hna6QkxelZiDOc5HnVGvQmEsLX&index=12&t=0s)]
+* David Sanders (Universidad Nacional Autónoma de México): _Rigorous global optimization in pure Julia_ [[notebook][david_sanders]] [[video](https://www.youtube.com/watch?v=D6BSXD5gMRk&list=PLP8iPy9hna6QkxelZiDOc5HnVGvQmEsLX&index=13&t=0s)]
```
Comment (mlubin) [inline _posts/2019-03-14-jump_dev_2019.md]: Same here

File: _posts/2019-03-14-jump_dev_2019.md [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+
+<tt>13:00 - 14:30</tt> **Keynotes**
+* Stefan Karpinski (Julia Computing): _The Unreasonable Effectiveness of Multiple Dispatch_ [[slides][stefan_karpinski]] [[video](https://www.youtube.com/watch?v=HAEgGFqbVkA&list=PLP8iPy9hna6QkxelZiDOc5HnVGvQmEsLX&index=2&t=0s)]
+* Miles Lubin (Google): _The roadmap for JuMP 1.0_ [[slides][miles_lubin]] [[video](https://www.youtube.com/watch?v=Lo9P9bD-JwI&list=PLP8iPy9hna6QkxelZiDOc5HnVGvQmEsLX&index=3&t=0s)]
+
+<tt>15:00 - 16:00</tt> **Energy**
+* Alessandro Soares (PSR): _A Julia/JuMP based Integrated Energy Resource Planning Model_ [[slides][alessandro_soares]] [[video](https://www.youtube.com/watch?v=rWmllzN4P18&list=PLP8iPy9hna6QkxelZiDOc5HnVGvQmEsLX&index=4&t=0s)]
+* Jose Daniel Lara (UC Berkeley / NREL): _PowerSimulations.jl a Integrated and Scalable Power Systems modeling tool_ [[notebook][jose_daniel_lara]] [[video](https://www.youtube.com/watch?v=JAHjZYiIJeI&list=PLP8iPy9hna6QkxelZiDOc5HnVGvQmEsLX&index=5&t=0s)]
```
Comment (mlubin) [inline _posts/2019-03-14-jump_dev_2019.md]: Another notebook linked to drive

**Pull request merged**